kit/inputs/src/UIMaterialTextView/hooks/useMaterialTextViewChildren.tsx

```diff
@@ -17,7 +17,29 @@ import type {
 } from '../types';
 
 const UIImageAnimated = Animated.createAnimatedComponent(UIImage);
-const UILabelAnimated = Animated.createAnimatedComponent(UILabel);
+
+function processChildren(
+    children: React.ReactNode,
+    tintColor: ColorVariants | undefined,
+): React.ReactNode {
+    return React.Children.map(children, (child: React.ReactNode) => {
+        if (typeof child === 'string') {
+            return (
+                <UILabel role={UILabelRoles.Action} color={tintColor}>
+                    {child}
+                </UILabel>
+            );
+        }
+        if (React.isValidElement(child) && child.type === UIImage) {
+            console.dir(child);
+            return React.createElement(UIImage, {
+                ...child.props,
+                tintColor,
```

Comment:
What do you think about specifying a size too?

Reply:
I'm not sure that this is necessary. We already have specified sizes in the UIImage component.

Reply:
Not sure that I understand what are you referring... Where exactly it's specified already?

Reply:
Let's provide the default sizes.